Add `osu! World Cup 2024 Concludes` news post (#12559)
ppy/osu-wiki: news/2024/2024-12-14-osu-world-cup-2024-concludes.md

---
layout: post
title: osu! World Cup 2024 Concludes
date: 2024-12-14 22:30:00 +0000
---

> They've all come so far, but there can only be one Champion.

Reviewer: capitalized champion looks weird there imo
```suggestion
They've all come so far, but there can only be one champion.
```

Author: i wrote it that way exactly because i wanted the capitalization on that word

Author: i got bullied into changing it https://github.com/ppy/osu-wiki/pull/12559/commits/c18604628bcfe913b20bdc203532ae1fcc3991c6

You can find a glimpse behind the scenes for the mapping and visuals of this tiebreaker in this video below:

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/AGjq63yxfog" frameborder="0" allowfullscreen></iframe>
</div>

> The three originals in the regular part of the pool were also very much worthy of attention. In the No Mod pool, there were two: [Naples Yellow Mango](https://osu.ppy.sh/beatmapsets/2288022#osu/4881869), by [Attoclef](https://osu.ppy.sh/beatmaps/artists/451) was a funky, ever-changing song made into an extremely tough gimmick map by the combined efforts of Chinese legends [fanzhen0019](https://osu.ppy.sh/users/418699) and [Hollow Wings](https://osu.ppy.sh/users/416662). [Corrupted Paywall](https://osu.ppy.sh/beatmapsets/2287942#osu/4881677), the frenetic speedcore track by [XH](https://osu.ppy.sh/beatmaps/artists/452), was adapted by [Cellina](https://osu.ppy.sh/users/2490770) to be a relentless speed challenge with a good amount of tech added on top. Finally, [XIchedelic Nova Trinity](https://osu.ppy.sh/beatmapsets/2287963#osu/4881714) was [DJ Genki](https://osu.ppy.sh/beatmaps/artists/295)'s contribution this year, with an electronic offering equal parts groovy and vibrant, mapped by [Pho](https://osu.ppy.sh/users/3624692) as a true test of aim and finger control in the Hidden pool.

Reviewer 2: ```suggestion
The three originals in the regular part of the pool were also very much worthy of attention. In the No Mod pool, there were two: [Naples Yellow Mango](https://osu.ppy.sh/beatmapsets/2288022#osu/4881869) by [Attoclef](https://osu.ppy.sh/beatmaps/artists/451) was a funky, ever-changing song made into an extremely tough gimmick map by the combined efforts of Chinese legends [fanzhen0019](https://osu.ppy.sh/users/418699) and [Hollow Wings](https://osu.ppy.sh/users/416662). [Corrupted Paywall](https://osu.ppy.sh/beatmapsets/2287942#osu/4881677), the frenetic speedcore track by [XH](https://osu.ppy.sh/beatmaps/artists/452), was adapted by [Cellina](https://osu.ppy.sh/users/2490770) to be a relentless speed challenge with a good amount of tech added on top. Finally, [XIchedelic Nova Trinity](https://osu.ppy.sh/beatmapsets/2287963#osu/4881714) was [DJ Genki](https://osu.ppy.sh/beatmaps/artists/295)'s contribution this year, with an electronic offering equal parts groovy and vibrant, mapped by [Pho](https://osu.ppy.sh/users/3624692) as a true test of aim and finger control in the Hidden pool.
```

im not the grammar guy but the comma feels weird

**[You can view the match VOD here!](https://www.twitch.tv/videos/2321357590)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/A_r3iiKN_gc?start=302&end=553" frameborder="0" allowfullscreen></iframe>
</div>

It was uncharted territory for the South Korean OWC roster. Though they'd played in the past two Grand Finals as the representative from the upper bracket, they had never been the favourites to win, even with the bracket reset advantage in their back pocket. That honour went to the United States, and with good reason: the U.S., after their 2023 triumph, had won eight of the last nine World Cups and were perennial champions.

2024, though, was different. Coming off their victory in the winners' bracket Final and playing at a level of historical dominance not seen since the famed 2019 USA roster, South Korea had finally flipped the script — the underdogs had become the favourites, with 67% of pick-em participants choosing them to defeat the six-time top podium defenders and to take home the titles of osu! Champions.

> Finally, match time arrived, and all discussions and analyses were cast aside as the 2024 OWC Grand Final got underway.

Reviewer 2: ```suggestion
Finally, match time arrived, and all discussions and analyses were cast aside as the 2024 OWC Grand Final was underway.
```

idk about "got" in this situation

| ::{ flag=CA }:: Canada vs. ::{ flag=GB }:: United Kingdom | Semifinals | [Text summary](https://osu.ppy.sh/home/news/2024-11-29-osu-world-cup-2024-semifinals-recap#canada-vs.--united-kingdom), [VOD](https://www.twitch.tv/videos/2310083959), [video recap](https://www.youtube.com/watch?v=udSXyASuYI4&t=393s) |
| ::{ flag=AU }:: Australia vs. ::{ flag=DE }:: Germany | Finals | [Text summary](https://osu.ppy.sh/home/news/2024-12-05-osu-world-cup-2024-finals-recap#australia-vs.--germany), [VOD](https://www.twitch.tv/videos/2315679372), [video recap](https://www.youtube.com/watch?v=-H-rOKG-2C4&t=436s) |
| ::{ flag=KR }:: South Korea vs. ::{ flag=US }:: United States | Finals | [Text summary](https://osu.ppy.sh/home/news/2024-12-05-osu-world-cup-2024-finals-recap#south-korea-vs.--united-states), [VOD](https://www.twitch.tv/videos/2315417594), [video recap](https://www.youtube.com/watch?v=-H-rOKG-2C4&t=645s) |

## osu! originals

<style>
    .osu-md--news .audio-player {
        max-width: inherit;
    }
</style>

> Alongside all the high-quality production and entertainment, the 2024 osu! World Cup also featured **13** brand-new [osu! originals](/wiki/osu!_originals), spanning across almost all major music genres. From jazz to hardcore, the quality of not only the songs, but also all other assets such as backgrounds, storyboards, videos and everything in between have helped generate hype and provide the community with content of the highest quality.

Reviewer 2: ```suggestion
Alongside all the high-quality production and entertainment, the 2024 osu! World Cup also featured **13** brand-new [osu! originals](/wiki/osu!_originals), spanning across almost all major music genres. From jazz to hardcore, the quality of not only the songs, but also all visual assets such as backgrounds, storyboards, videos and everything in between have helped generate hype and provide the community with content of the highest quality.
```

...assuming it's all visuals. this leaves me wondering what "other" means beyond what's listed lol

Reviewer 3: as in graphic design, mapping and perhaps other minor things like sound effects or lyrics (where "everything in between" can cover it all)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/451/Songs/Attoclef%20-%20Naples%20Yellow%20Mango.mp3">
    </audio>
</div>

[![](/wiki/shared/news/2024-12-14-osu-world-cup-2024-concludes/Corrupted_Paywall-banner.jpg)](https://osu.ppy.sh/beatmapsets/2287942#osu/4881677)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/452/Songs/XH%20-%20Corrupted%20Paywall.mp3">
    </audio>
</div>

[![](/wiki/shared/news/2024-12-14-osu-world-cup-2024-concludes/XIchedelic_Nova_Trinity-banner.jpg)](https://osu.ppy.sh/beatmapsets/2287963#osu/4881714)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/295/Songs/Gram_vs._DJ_Genki_-_XIchedelic_Nova_Trinity.mp3">
    </audio>
</div>

[![](/wiki/shared/news/2024-12-14-osu-world-cup-2024-concludes/Operation_Zenithfall-banner.jpg)](https://osu.ppy.sh/beatmapsets/2287992#osu/4881796)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/31/Songs/Camellia_-_Operation_Zenithfall.mp3">
    </audio>
</div>

## Statistics

> We've compiled some interesting stats for you to look at.

Reviewer: ```suggestion
Let's go through some interesting stats that we've compiled for your viewing pleasure:
```

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9224078">FlyingTuna</a></p>

Yes, Nopekjk was our analyst. Throughout the whole OWC he would analyze the scores that the opponents would get on each map of each stage and would help us with pick and ban whilst also encouraging us and supporting us mentally.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3958619" style="background-image: url('/wiki/shared/avatars/dio.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3958619">D I O</a></p>

The Grand Final had a turning point when South Korea was down 4–1 early on. How did your team react, and what changed in order for you to win?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9224078" style="background-image: url('/wiki/shared/avatars/FlyingTuna.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9224078">FlyingTuna</a></p>

> My team and I, we all took it calmly as we had the bracket reset in mind. Because no matter what happens in the first part, even if we fail we had that 1 more chance. So we all remained calm and positive and even if we were down 4-1 we were just as confident and strong as ever because we are used to reverse sweeping.

Reviewer 4: ```suggestion
My team and I, we all took it calmly as we had the bracket reset in mind. Because no matter what happens in the first part, even if we fail we had that one more chance. So we all remained calm and positive and even if we were down 4-1 we were just as confident and strong as ever because we are used to reverse sweeping.
```

Reviewer 3: eh

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3958619">D I O</a></p>

South Korea finished in the top 4 eight times since 2014, top 3 5 times since 2016, and second in 2019, 2022, and 2023. What does it mean for you to finally get the win this year?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9224078" style="background-image: url('/wiki/shared/avatars/FlyingTuna.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9224078">FlyingTuna</a></p>

To be honest I can't really express my emotions enough with just words. I am so overjoyed and happy that this happened after everything we as a team have been through. This means so much not just to the 2024 roster players but also to past roster players who have been cheering and supporting us throughout all this time. It is definitely one of the greatest things in my life and I will cherish it always.

</div>

::{#alexdunk}::

![](/wiki/shared/news/2024-12-14-osu-world-cup-2024-concludes/AlexDunk-banner.png)

> For OWC 2024, [AlexDunk](https://osu.ppy.sh/users/2287992) was responsible for the graphic designs of [AJITAMA Adventure OST](https://osu.ppy.sh/beatmapsets/2274953), [Whiteout](https://osu.ppy.sh/beatmapsets/2278209), [Naples Yellow Mango](https://osu.ppy.sh/beatmapsets/2288022) and [Corrupted Paywall](https://osu.ppy.sh/beatmapsets/2287942). These were, however, just four among 13 osu! original projects in AlexDunk's 2024 world cup (and [LGA](/wiki/Tournaments/LGA/2024)) portfolio.

Reviewer 4: ```suggestion
For OWC 2024, [AlexDunk](https://osu.ppy.sh/users/9194799) was responsible for the graphic designs of [AJITAMA Adventure OST](https://osu.ppy.sh/beatmapsets/2274953), [Whiteout](https://osu.ppy.sh/beatmapsets/2278209), [Naples Yellow Mango](https://osu.ppy.sh/beatmapsets/2288022) and [Corrupted Paywall](https://osu.ppy.sh/beatmapsets/2287942). These were, however, just four among 13 osu! original projects in AlexDunk's 2024 world cup (and [LGA](/wiki/Tournaments/LGA/2024)) portfolio.
```

---

<div class="news-chat-quote__text-container">

> <a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2287992" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>

Reviewer 4: ```suggestion
<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5773079" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>
```

> <p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/2287992">Walavouchey</a></p>

Reviewer 4: ```suggestion
<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/5773079">Walavouchey</a></p>
```

How would you describe yourself to someone who doesn't know you?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9194799" style="background-image: url('/wiki/shared/avatars/AlexDunk.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9194799">AlexDunk</a></p>

Hi, thanks for having me! I go by AlexDunk, originally an osu!mania player and mapper at heart. But now I'm slowly transitioning to be doing graphic design for song backgrounds! I have always been enjoying making beatmap backgrounds for songs, and the background images of all of my uploaded maps since 2019 are edited through Photoshop, so being able to join the World Cup design team for the same objectives is honestly such a honor. My goal is to keep making bigger and more awesome works that people enjoy to look at, and at the same time mayyybe inspire people too to start their own creative hobby and just simply doing awesome things!

> <a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2287992" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>

Reviewer 4: ```suggestion
<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5773079" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>
```

> <p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/2287992">Walavouchey</a></p>

Reviewer 4: ```suggestion
<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/5773079">Walavouchey</a></p>
```

To showcase my commitment to this project, I decided to implement Blender (A popular 3D software) for the first time in my background art design. It turned out amazing and I'm still really proud of it.

Other than that, the CWC 2024 Grand Finals tiebreaker: [Frost Walker](https://osu.ppy.sh/beatmapsets/2202823) by [Kou!](https://osu.ppy.sh/beatmaps/artists/383) is such an enormous production from start to finish. I am very grateful to even be able to be part of it and contribute to this project (Big shout out to [Sakura006](https://osu.ppy.sh/users/10365024) and the illustrators involved ([Dreamxiety](https://osu.ppy.sh/users/13103233), [Saturn](https://osu.ppy.sh/users/22001295), [Nepperoni](https://osu.ppy.sh/users/7082924)) for the carry).

> <a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2287992" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>

Reviewer 4: ```suggestion
<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5773079" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>
```

> <p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/2287992">Walavouchey</a></p>

Reviewer 4: ```suggestion
<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/5773079">Walavouchey</a></p>
```

Are there early concepts or interesting stories that you remember well?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9194799" style="background-image: url('/wiki/shared/avatars/AlexDunk.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9194799">AlexDunk</a></p>

I remember my journey into GFX began in 2018, back when I was still deeply obsessed with grinding osu!mania ranks. One day I stumbled upon a [beatmap for "Light It Up"](https://osu.ppy.sh/beatmapsets/455164) by [Hiranaka](https://osu.ppy.sh/users/4207811) and was immediately captivated by the typography and the overall mood of the background. Its simplicity perfectly complemented the song, inspiring me to create something similar for my own maps.

That moment sparked my interest in GFX, starting with basic text overlays and gradually evolving into adding effects, crafting detailed typography, and eventually producing full-fledged Photoshop artworks. Today, I have the privilege of collaborating with talented individuals who share my passion, working on projects I could never have imagined when I first began this journey.

> <a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2287992" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>

Reviewer 4: ```suggestion
<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5773079" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>
```

> <p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/2287992">Walavouchey</a></p>

Reviewer 4: ```suggestion
<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/5773079">Walavouchey</a></p>
```

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9194799">AlexDunk</a></p>

Working with artists on a design project is new to me too. I've only grown comfortable with it after participating collabs time and time again. Obviously you would want to make sure you have active presence, sharing your thoughts / ideas together with the team, but in the end it still comes down to two things (let's assume you are a GFX artist working with illustrator(s)):

1. Your contribution to the final work: Be proactive and ensure your input is valuable. Collaborate openly, but don't hesitate to take initiative—your unique touch as a GFX artist matters. Strive to leave a mark that aligns with the team's vision.
2. Keep time constraint in mind, plan things out and finish them bit by bit. Break tasks into manageable parts and stick to a timeline to avoid rushing. Consistent progress ensures you deliver quality work without compromising the team's overall flow.

Doing and learning at the same time will always push you further.

> <a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2287992" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>

Reviewer 4: ```suggestion
<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5773079" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>
```

> <p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/2287992">Walavouchey</a></p>

Reviewer 4: ```suggestion
<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/5773079">Walavouchey</a></p>
```

Looking ahead, I'm excited to take on the responsibilities of making main design for the World Cups 2025, working alongside the talented illustrator [Pineapple](https://osu.ppy.sh/users/11523326)—whom I admire as their #1 fan. This marks my first time designing for tournaments on this scale, but I'm committed to delivering a presentation worthy of such an event.

While I can't reveal more projects of what I have been busy with just yet, please stay tuned for the upcoming designs. And I promise, they are going to be bangers.

</div>

::{#camellia}::

![](/wiki/shared/news/2024-12-14-osu-world-cup-2024-concludes/Camellia-banner.png)

Likely needing no introduction, [Camellia](https://osu.ppy.sh/beatmaps/artists/31) is an artist responsible for many rhythm gamers' broken fingers, as well as many songs with storied beatmaps here on osu!. This year, he composed *Operation: Zenithfall* for the osu! World Cup's [Grand Finals tiebreaker beatmap](https://osu.ppy.sh/beatmapsets/2287992).

---

<div class="news-chat-quote__text-container">

> <a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2287992" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>

Reviewer 4: ```suggestion
<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5773079" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>
```

> <p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/2287992">Walavouchey</a></p>

Reviewer 4: ```suggestion
<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/5773079">Walavouchey</a></p>
```

How would you describe the concept or inspirations behind *Operation: Zenithfall*?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/beatmaps/artists/31" style="background-image: url('/wiki/shared/avatars/Camellia.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-fa" href="https://osu.ppy.sh/beatmaps/artists/31">Camellia</a></p>

"Operation: Zenithfall" was created as the track for the tournament's - OWC2024 - absolute finale. The theme is, very coincidentally, based on the idea of "overturning the top - Zenith", expressing the players' challenges and the drama of turning the tables through music. This track was not only to capture intensity and epicness, but also to deliver a unique experience unlike any other song.

> During the brainstorming with concept idea process, I thought, "What if the theme were about carrying out a mission as a member of a secret, covert organization?" By aligning that concept with the players' experience, especially for the attendees/players of the tournament, I decided to incorporate that idea into the music in a curveball way. So I drew inspiration from the iconic sounds of spy films, particularly elements reminiscent of 007. This style of music is rarely heard in the rhythm game scene, so I believe it brings a refreshing and entertaining quality.

Reviewer 3: we don't need to turn interview responses into british english

I still vividly remember the excitement and honor I felt when the track was actually played as a tiebreaker in the tournament - it was a very intense and exciting moment for me, and hopefully for the most other people watching, or playing, too. It's absolutely honorful to see that people still love the track with passion, and I believe it's genuinely from the magic that the rhythm games have.

What makes me even happier is how this track became the starting point for a tradition of original songs being created for osu! tournaments. The fact that history has been shaped, even in a small way, by something I created is an incredible honor, and I'm proud to have been part of that journey. It's truly amazing to see how far things and other artists' works have come since then, and I'm thankful to have had the chance to contribute to it.

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2287992" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>

> <p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/2287992">Walavouchey</a></p>

Reviewer 4: ```suggestion
<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/5773079">Walavouchey</a></p>
```

When I first became involved with the rhythm game sphere - it was an arcade rhythm game title SOUND VOLTEX by KONAMI - it was simply a way for me to share my music, and share my love and creation. I was already a huge fan of rhythm games as a player, completely obsessed with their fun, but I never imagined that my tracks would someday be regarded as "boss songs" within them.

In the beginning, I focused on creating music that expressed my favorite genres and styles, enjoying the process of seeing how my work would be used in games and received by players.

But over time, I began to embrace the idea of "creating music that enhances the gameplay experience," consciously incorporating intense rhythm parts and challenging elements unique to rhythm games. As a result, I'm incredibly happy that many players have enjoyed my music, and that it's often recognized as "boss songs" as their biggest challenges. I could never have dreamed of becoming as deeply involved with the rhythm game community as I am today, but it has been one of the greatest joys and honors of my career.

> <a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2287992" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>

Reviewer 4: ```suggestion
<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5773079" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>
```

> <p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/2287992">Walavouchey</a></p>

Reviewer 4: ```suggestion
<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/5773079">Walavouchey</a></p>
```

Is there a story behind your fedora?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/beatmaps/artists/31" style="background-image: url('/wiki/shared/avatars/Camellia.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-fa" href="https://osu.ppy.sh/beatmaps/artists/31">Camellia</a></p>

Nowadays, it's not just something I wear on stage - it's become a part of me, completely, like glasses. Thanks to that, many people remember me as "the guy with the fedora", and I really like that! These days, I almost never leave the house without a hat, and honestly, the idea of going out without one is getting a bit scary - like I feel like being naked...

> Even people who have never met me in person recognize me by my hairstyle, glasses, and, of course, this fedora, which makes meeting up incredibly easy. It's also not uncommon for people to spot me in real life and ask, "Are you Camellia?". That's insane!

Reviewer 4: ```suggestion
Even people who have never met me in person recognise me by my hairstyle, glasses, and, of course, this fedora, which makes meeting up incredibly easy. It's also not uncommon for people to spot me in real life and ask, "Are you Camellia?". That's insane!
```

> <a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/2287992" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>

Reviewer 4: ```suggestion
<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5773079" style="background-image: url('/wiki/shared/avatars/Walavouchey.jpg')"></a>
```

> <p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/2287992">Walavouchey</a></p>

Reviewer 4: ```suggestion
<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/5773079">Walavouchey</a></p>
```

Do you have any general thoughts to share about this project and the people you've been working with?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/beatmaps/artists/31" style="background-image: url('/wiki/shared/avatars/Camellia.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-fa" href="https://osu.ppy.sh/beatmaps/artists/31">Camellia</a></p>

The people I've been working with are incredibly talented, passionate, and extremely kind to me and everyone. Every time I get to experience their energy and creativity, it inspires me a lot. Additionally, I'm so happy to be back here after four years. Though I've worked with mania, taiko and catch mode tournaments too, seeing people getting happy that I'm back is amazing.

> While working on the music, I was able to incorporate their feedback and ideas, which I believe made the final product even better. I want to express my deepest gratitude to everyone involved, players participating in the tournament, organizers, staff, and everyone else who contributed. Thanks to all your efforts and support, this project has come to life. It was absolute cinema!

Reviewer 4: ```suggestion
While working on the music, I was able to incorporate their feedback and ideas, which I believe made the final product even better. I want to express my deepest gratitude to everyone involved, players participating in the tournament, organisers, staff, and everyone else who contributed. Thanks to all your efforts and support, this project has come to life. It was absolute cinema!
```
